kshebeko/lesson4/src/com/company/kshebeko_class/Flower.java

```diff
@@ -0,0 +1,32 @@
+package com.company.kshebeko_class;
+
+public class Flower {
+    public String name;
+    public String accessories;
+    public final int price = 30;
+    public static int flowers_in_bouquet = 0;
```

Comment:
in java usually camel case is used for naming - https://ru.wikipedia.org/wiki/CamelCase
CamelCase - for classes, lowerCamelCase - for variables and methods